evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -909,23 +917,42 @@ def test_generate_pipeline_example(
         trans_primitives=["absolute"],
         return_types="all",
     )
+
+    X_train_t.index = y_train.index
     features_path = os.path.join(str(tmpdir), "features.json")
     ft.save_features(features, features_path)
-
+    ensembling = ensembling if not is_time_series(automl_type) else False
     aml = AutoMLSearch(
         X_train=X_train_t,
         y_train=y_train,
         problem_type=automl_type,
         optimize_thresholds=False,
-        max_iterations=5,
         problem_configuration=problem_configuration,
         features=features,
+        ensembling=ensembling,
     )
-    env = AutoMLTestEnv(automl_type)
-    with env.test_context(score_return_value={aml.objective.name: 1.0}):
-        aml.search()
 
-    pipeline = aml.get_pipeline(2)
+    if ensembling:
```

Comment:
Ensembling pipelines do not work as each select columns transformer is not fit and will return no columns when using the test env. This also fails for any catboost or xgboost pipeline. Will file an issue to patch this behavior in `AutoMLTestEnv` but wanted to get this out first.